src/Microsoft.DotNet.Interactive.Journey/KernelExtensions.cs

```diff
@@ -37,18 +37,18 @@ public static CompositeKernel UseModelAnswerValidation(this CompositeKernel kern
 
         var fromUrlOption = new Option<Uri>(
             "--from-url",
-            "Specify lesson source URL");
+            Resources.CompositeKernelFromUrlOptionDescription);
```

Comment:
Not sure about the resource name `CompositeKernelFromUrlOptionDescription` to eval to `Specify lesson source URL` strings. Should this be something along the line of `LessonFromUrlOptionDescription` (this should probably be `journey` instead of `lessons`)